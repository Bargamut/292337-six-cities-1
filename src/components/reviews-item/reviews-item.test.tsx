import * as React from 'react';
import * as renderer from 'react-test-renderer';

import ReviewsItem from './reviews-item';

const mocks = {
  comment: {
    id: 1,
    user: {
      id: 2,
      isPro: false,
      name: ``,
      avatarUrl: `4.jpg`
    },
    rating: 2,
    comment: `The room was spacious and clean.`,
    date: `2019-06-20T16:39:57.287Z`
  }
};

it(`Property correctly renders after relaunch`, () => {
  const { comment } = mocks;

  const property = renderer.create(
    <ReviewsItem item={comment} />
  ).toJSON();

  expect(property).toMatchSnapshot();
});
