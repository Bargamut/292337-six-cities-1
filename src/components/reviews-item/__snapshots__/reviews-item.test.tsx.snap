// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property correctly renders after relaunch 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="4.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "40%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      The room was spacious and clean.
    </p>
    <time
      className="reviews__time"
      dateTime="2019-06-20T16:39:57.287Z"
    >
      2019-6-20
    </time>
  </div>
</li>
`;
